### PR TITLE
Add density units used in clinical chemistry.

### PR DESCRIPTION
**Pull request by JonathanDavies626**
Add density units used in clinical chemistry (g/L, g/dL and g/mL).

### CODE REVIEW

File: UnitsNet/UnitsNet.WindowsRuntimeComponent-Net45.csproj [hunk 1]
```diff
@@ -53,7 +53,186 @@ There is probably a better way to achieve this?
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs;" Exclude="obj\**;Properties\AssemblyInfo.cs" />
```
Comment (anjdreas) [inline UnitsNet/UnitsNet.WindowsRuntimeComponent-Net45.csproj]: This happens often, but please revert all changes to `.csproj` files. Only the wildcard include should remain.

File: UnitsNet/UnitDefinitions/Density.json [hunk 2]
```diff
@@ -122,6 +122,45 @@
...
+      "Prefixes": [ "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/L" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "GramPerDeciLiter",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/Density.json]: Lowercase `l` in `GramPerDeciliter`.

File: UnitsNet/UnitDefinitions/Density.json [hunk 3]
```diff
@@ -122,6 +122,45 @@
...
+      "Prefixes": [ "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/dL" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "GramPerMilliLiter",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/Density.json]: Lowercase `l` in `GramPerDeciliter`.

File: UnitsNet/UnitDefinitions/Density.json [hunk 4]
```diff
@@ -122,6 +122,45 @@
...
+      "PluralName": "GramsPerMilliLiter",
+      "FromUnitToBaseFunc": "x/1e-3",
+      "FromBaseToUnitFunc": "x*1e-3",
+      "Prefixes": [ "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/mL" ]
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/Density.json]: Lowercase `l` in `ml`. That is the convention we use for liters in [Volume](https://github.com/anjdreas/UnitsNet/blob/master/UnitsNet/UnitDefinitions/Volume.json).

File: UnitsNet/UnitDefinitions/Density.json [hunk 5]
```diff
@@ -122,6 +122,45 @@
...
+    {
+      "SingularName": "GramPerDeciLiter",
+      "PluralName": "GramsPerDeciLiter",
+      "FromUnitToBaseFunc": "x/1e-1",
+      "FromBaseToUnitFunc": "x*1e-1",
+      "Prefixes": [ "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/dL" ]
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/Density.json]: Lowercase `l` in `ml`. That is the convention we use for liters in [Volume](https://github.com/anjdreas/UnitsNet/blob/master/UnitsNet/UnitDefinitions/Volume.json).

File: UnitsNet/CustomCode/UnitClasses/Density.extra.cs [hunk 6]
```diff
@@ -20,28 +20,61 @@
...
     public partial struct Density
+#endif
     {
+
+#if WINDOWS_UWP
+        internal
+#else
+        public
+#endif
+            Density(Molarity molarity, Mass molecularWeight)
```
Comment (anjdreas) [inline UnitsNet/CustomCode/UnitClasses/Density.extra.cs]: I don't think we should add a constructor for this. The convention is for the constructor to set the base unit value, so instead the `FromMolarity()` method below should call the default generated constructor with the kg/m³ value you calculate here, or use the `FromKilogramsPerCubicmeter()` constructor method.

File: UnitsNet/CustomCode/Extensions/DensityExtensions.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+namespace UnitsNet.CustomCode.Extensions
+{
+    /// <summary>
+    ///     Extension methods for <see cref="Density" />.
+    /// </summary>
+    public static class DensityExtensions
```
Comment (anjdreas) [inline UnitsNet/CustomCode/Extensions/DensityExtensions.cs]: Nice that you figured out the convention of using extension classes, I don't think I documented that in the wiki steps.

**Comment by JonathanDavies626** [comment]
Hi Andreas, I've pushed another commit to address the issues you raised.

**Comment by anjdreas** [comment]
Perfect, thanks.

**Comment by anjdreas** [comment]
Nuget 3.62 is out.

**Comment by JonathanDavies626** [comment]
That's great. Thanks Andreas.

**Pull request merged**